core/src/test/java/org/apache/iceberg/TestReplacePartitions.java

```diff
@@ -67,6 +67,34 @@ public class TestReplacePartitions extends TestBase {
           .withRecordCount(1)
           .build();
 
+  static final DataFile FILE_NULL_PARTITION =
+      DataFiles.builder(SPEC)
+          .withPath("/path/to/data-null-partition.parquet")
+          .withFileSizeInBytes(0)
+          .withPartitionPath("data_bucket=__HIVE_DEFAULT_PARTITION__")
+          .withRecordCount(0)
+          .build();
+
+  // Partition spec with VOID partition transform ("alwaysNull" in Java code.)
+  public static final PartitionSpec SPEC_VOID =
```

Comment:
```suggestion
 static final PartitionSpec SPEC_VOID =
```

Reply:
Since it is in a test, I'll go ahead and merge this 👍